core/shared/src/main/scala/zio/ZLayer.scala

```diff
@@ -2039,6 +2045,14 @@ object ZLayer {
      */
     def update[A: Tagged](f: A => A)(implicit ev: ROut <:< Has[A]): ZLayer[RIn, E, ROut] =
       self >>> ZLayer.fromFunctionMany(_.update[A](f))
+
+    /**
+      * A named alias for `++`.
+      */
+    def zip[E1 >: E, RIn2, ROut1 >: ROut, ROut2 <: Has[_]](
```

Comment:
How about `and` or maybe `combine`?

`Clock.live and System.live and (HttpClient.live injectInto StorageAPI.live)`

Reply:
Sure. We could also do `to` for `>>>` so they were both short.

Reply:
👍

Reply:
Done.